### PR TITLE
Feature/sdk 197 Update integration docs for demo and dev repos

### PR DESCRIPTION
**Pull request by blaZ3**
PR for the on-going revised draft of SDK documentation for clients.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,33 @@
...
+* [Going live](#going-live)
+* [Releases](#releases)
+* [Upgrading from SDK 1.* to 2.+](#upgrading-sdk)
+
+## Overview
+
+The Android SDK allows you to add the Veriff verification flow to your native Android application.
+The documentation for Veriff backend integration can be found [here](https://developers.veriff.me/).
+
+## Adding the sdk
```
Comment (madisp) [inline README.md]: nit: sdk -> SDK

File: README.md [hunk 2]
```diff
@@ -24,93 +37,127 @@ Add two new maven repository destination in the project-level build.gradle repos
...
-  Veriff SDK versions 1.* are no longer being supported, please integrate version 2.* instead.
+The verification process must be launched inside the vendor specific Activity class.
+The activity is aware of the current customer, who is either logged in via the Vendor system or who
+is identified other way with unique system pass. The vendor must be able to determine the customer
+later, if application ends/ or returns. It depends entirely on the Vendor business logic.
 
-  To integrate Veriff into a new project, follow the previous section
-  To upgrade from 1.* to 2.* refer to section [on upgrading Veriff](#upgrading-veriff)
+Every Veriff SDK session is unique for a client. The session expires after 7 days automatically,
+but is valid until then. After the verification data is uploaded, the SDK v1.0 does not wait for the
```
Comment (madisp) [inline README.md]: what's SDK v1.0 in this context?

File: README.md [hunk 3]
```diff
@@ -24,93 +37,127 @@ Add two new maven repository destination in the project-level build.gradle repos
...
+final verification result (async). The SDK v1.0 only notifies whether the verification data upload is successful or not.
 
-## Using the Veriff SDK
+In vendor activity class define the result code and initialize the SDK and launch the verification
+flow as below:
 
-### 2. Initializing SDK in project code
+| Parameters   | Description                                                  |
+| ------------ | ------------------------------------------------------------ |
+| baseUrl      | 'https://staging.veriff.me/v1/' for testing and 'https://magic.veriff.me/v1/' for production |
```
Comment (madisp) [inline README.md]: :-1: the URL's shouldn't include `/v1/`, we actually actively discourage that at runtime by emitting a warning in logcat

File: README.md [hunk 4]
```diff
@@ -24,93 +37,127 @@ Add two new maven repository destination in the project-level build.gradle repos
...
 ```
 
-  After the service is set up then it needs to be declared in the **application manifest**:
-```
+Add the service to the application manifest:
+
+```xml
 <!-- Implement Veriff SDK updates service -->
-<service android:name=".service.VeriffSDKStatusUpdatesService">
+<service android:name=".MyVeriffStatusUpdatesService">
```
Comment (madisp) [inline README.md]: unrelated to this commit, but oof, doesn't this export the service? :scream:

File: README.md [hunk 5]
```diff
@@ -24,93 +37,127 @@ Add two new maven repository destination in the project-level build.gradle repos
 
         }
     }
-  ```
-
-Add two dependency imports in the application build.gradle dependency list:
-
-  ``` java
+```
+Add two dependencies in the application ```build.gradle``` :
```
Comment (madisp) [inline README.md]: nit: remove extra space after `build.gradle`

File: README.md [hunk 6]
```diff
@@ -136,185 +183,23 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
...
 
-# Upgrading Veriff
-
-### 3. Upgrading Veriff SDK from 1.* to 2.*
-
-Veriff SDK 2.* integration has changed significantly since 1.*. There are major changes in the SDK distribution where the SDK from now on is distributed via gradle import and many previously required services have been removed and trimmed down.
+## Adding error logging
 
-## 3.1 Remove Veriff SDK required Firebase code
+To turn on logging, you simply add your logging implementation instance(instance of LogAccess class)to
```
Comment (madisp) [inline README.md]: nit: missing spaces around the parentheses

File: README.md [hunk 7]
```diff
@@ -420,3 +305,114 @@ if (!handleVeriffNotifications(remoteMessage)) {
 
 -dontwarn com.koushikdutta.ion.conscrypt.ConscryptMiddleware
 ```
+
+## Going live
+
+Set **baseUrl** for initialising the SDK as 'https://magic.veriff.me/v1/'
```
Comment (madisp) [inline README.md]: this section is a bit weird - we already have the staging vs prod URL above

Comment (madisp) [reply README.md]: also, URL should be withouth the `/v1/` path

File: README.md [hunk 8]
```diff
@@ -420,3 +305,114 @@ if (!handleVeriffNotifications(remoteMessage)) {
...
+
+  Remove irrelevant return values from VeriffSDKStatusUpdatedService and VeriffStatusReceiver
+
+``` java
+  VeriffConstants.STATUS_OUT_OF_BUSINESS_HOURS, VeriffConstants.STATUS_UNABLE_TO_RECORD_AUDIO and VeriffConstants.STATUS_VIDEO_CALL_ENDED
+```
+  As a final step add the import for Veriff libary in the application build.gradle dependency list. It should contain the following two lines:
+
+``` java
+  implementation 'com.veriff:veriff-library:2.1.1'
```
Comment (madisp) [inline README.md]: :-1: Change 2.1.1 to 2.3.0

**Review by madisp — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (blaZ3) [reply README.md]: Yes it will make it exported, I think we should make it exported=false, since no one from outside should be able to call it right?

Comment (blaZ3) [reply README.md]: Its exported=false in the sample app manifest LOL

Comment (madisp) [reply README.md]: no need to change it here, I'll file a ticket in JIRA

**Review by madisp — APPROVED** [review]

**Pull request merged**